sdk/identity/identity/README.md

```diff
@@ -8,9 +8,42 @@ to authenticate API requests. It supports token authentication using an Azure Ac
 
 ### Prerequisites
 
-- an Azure subscription
-  - if you don't have one, you can sign up for a [free account](https://azure.microsoft.com/free/)
 - Node.js 8 LTS or higher
+- An Azure subscription.
+    - You can sign up for a [free account](https://azure.microsoft.com/free/).
+- The [Azure CLI][azure_cli] can also be useful for authenticating in a development environment, creating accounts, and managing account roles.
+
+#### Authenticating via Visual Studio 
+
+Developers using Visual Studio 2017 or later can authenticate an Azure Active Directory account through the IDE. Applications using the `DefaultAzureCredential` or the `VisualStudioCredential` can then use this account to authenticate calls in their application when running locally.
+
+To authenticate in Visual Studio select the `Tools > Options` menu to launch the Options dialog. Then navigate to the `Azure Service Authentication` options to sign in with your Azure Active Directory account.
+
+![Visual Studio Account Selection](./images/VsLoginDialog.png)
```

Comment:
I wouldn't add this to your readme, for JS/TS just VSCode and Azure CLI should be called out.

Reply:
thank you!